src/us/kbase/workspace/test/kbase/KBaseWorkspaceConfigTest.java

```diff
@@ -166,7 +468,7 @@ public void configMaximalS3() throws Exception {
 				.with("backend-token", "    token token token    ")
 				.with("backend-container", "   mahbukkit   ")
 				.with("backend-region", "   a-lovely-region   ")
-				.with("backend-trust-all-ssl-certificates", "trudat")
+				.with("backend-trust-all-ssl-certificates", "     true    ")
```

Comment:
So it turns out that a true case was never tested for this config item. Whoops. Shows the value of compiler warnings I guess